control-plane/agents/core/src/volume/tests.rs

```diff
@@ -569,15 +569,20 @@ async fn nexus_persistence_test(cluster: &Cluster) {
 }
 async fn nexus_persistence_test_iteration(local: &NodeId, remote: &NodeId, fault: FaultTest) {
     tracing::debug!("arguments ({:?}, {:?}, {:?})", local, remote, fault);
-
+    let mut allowed_nodes: Vec<String> = Vec::new();
+    let preferred_nodes: Vec<String> = Vec::new();
+    allowed_nodes.push(String::from(local.clone()));
```

Comment:
you can simplify things here, to get a String from NodeId you can use .to_string()
you can also declare the vector in one go, eg: let allowed_nodes = vec![ local.to_string(), remote.to_string()]